### DISCUSSION TITLE
Large Google Import - Duration

### DISCUSSION THREAD

**Discussion by rantanlan** [Freika/dawarich#115] [Q&A] [answered]
Just tried to import my 12 year location history (4,5 million points), import of the points took 3 days. Now it seems it is trying to reverse geolocate every point which, for my calculation, will take several weeks? Is this needed? Not sure what to expect here… any experiences from you guys? It looks like the history will only show it when the import is finished?

**Comment by Freika** [comment, marked as answer]
The importing process is not tied to the reverse geocoding process. As soon as the importing process is finished, points will appear on the map.

**Reply by rantanlan** [reply]
Thanks for the info, maybe I have an issue with the import then, since zero points are showing up.
Edit: Okay if i search for dates, the points show up.. was just irritated that there where no years or anything shown initialy

**Reply by Freika** [reply]
@rantanlan if you have ideas on how to improve the interface, feel free to reach me in Discord (link to server in readme) and we can discuss it there :)
